tests/276_Imagick_artifacts.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+Test artifacts
+--SKIPIF--
+<?php
+
+require_once(dirname(__FILE__) . '/skipif.inc');
+checkClassMethods('Imagick', array('setImageArtifact', 'getImageArtifact', 'deleteImageArtifact'));
+
+/* See imagick_class.c */
+if (Imagick::getVersion()['versionNumber'] < 0x656) {
```

Comment:
So, why does it work with the foo()['..'] syntax over here? And why doesn't the checkClassMethods() check work, i.e. why is there an error that the method does not exist despite the check?